m4/check_pkg_lttng.m4

```diff
@@ -35,7 +35,7 @@ AC_DEFUN([CHECK_PKG_LTTNG], [
          LIBS="${check_pkg_LIBS_save}"
          $2])
 
-  AC_DEFINE_UNQUOTED([HAVE_LIBLTTNG_UST], [${check_pkg_found}], [Defined to 1 if lttng-ust is requested and available])
+  AC_DEFINE_UNQUOTED([HAVE_LIBLTTNG_UST], [${check_pkg_define}], [Defined to 1 if lttng-ust is requested and available])
```

Comment:
this typo is indeed interesting. so, "yes" -> "1" for HAVE_LIBLTTNG_UST.
Question: do we see LTTNG output being actually generated with this patch now?

Reply:
Yes, I confirmed that LTTNG traces are generated with this PR.